paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AddressLauncher.kt

```diff
@@ -183,4 +200,21 @@ internal class AddressLauncher internal constructor(
             REQUIRED
         }
     }
+
+    companion object {
+        @Composable
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun rememberLauncher(
+            callback: AddressLauncherResultCallback
+        ): AddressLauncher {
+            val activityResultLauncher = rememberLauncherForActivityResult(
+                contract = AddressElementActivityContract(),
+                onResult = callback::onAddressLauncherResult
+            )
+
+            return remember {
+                AddressLauncher(activityResultLauncher)
+            }
+        }
+    }
```

Comment:
This seems unused. Can we remove it?

Reply:
This is for public use, I forgot to remove the annotation

Reply:
What are your thoughts on exposing this as a top-level `rememberAddressLauncher()` method instead of a companion method? Feels more in line with the typical Compose approach.